packages/dialtone-icons/src/keywords-icons.json

```diff
@@ -489,7 +489,23 @@
       ],
       "whisper": [
         "soft"
-      ]
+      ],
+      "power-dialer": [
+        "powerdialer",
+        "dialer",
```

Comment:
Redundant. Don't repeat words from the icon name since those will be found in searching anyway. Ask @bhawkins-dialpad for additional relevant keywords.

Reply:
@fede-dp @francisrupert Ok lets just keep "powerdialer" as long as folks are able to find it using "dialer" as that is commonly used too.